content/guides/installing/making_haiku_usb_stick.html

```diff
@@ -67,17 +67,17 @@
 
 <h1>Windows</h1>
 
-<h2>Using ImageWriter for Windows</h2>
+<h2>Using Win32DiskImager for Windows</h2>
 
-<div class="alert alert-info">Note that ImageWriter for Windows is considered experimental (alpha) software by it's own author. Be sure to read the README.txt file included with the software.</div> 
+<div class="alert alert-info">Note that Win32DiskImager for Windows is considered beta software by it's own author. Be sure to read the README.txt file included with the software.</div> 
 
 <p>Rename the downloaded haiku-anyboot.image to haiku-anyboot.img otherwise you won't see it in ImageWriter's file selection panel.</p>
 
-<p>Get ImageWriter for Windows (https://launchpad.net/win32-image-writer). Unzip it in the place of your choice.</p>
+<p>Get Win32DiskImager for Windows (https://sourceforge.net/projects/win32diskimager/). After you download it, runs the simple installation program.</p>
```

Comment:
Sorry for the late review...
Can you linkify the URL ("href") and while you're at it, other URLs on that page?